ui/hooks/useIsOriginalNativeTokenSymbol.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import fetchWithCache from '../../shared/lib/fetch-with-cache';
+
+export function useIsOriginalNativeTokenSymbol(chainId, ticker) {
+  const [isOriginalNativeSymbol, setIsOriginalNativeSymbol] = useState(null);
+
+  useEffect(() => {
+    async function getNativeTokenSymbol(networkId) {
+      try {
+        const safeChainsList = await fetchWithCache({
+          url: 'https://chainid.network/chains.json',
+          functionName: 'getSafeChainsList',
+        });
+
+        const matchedChain = safeChainsList.find(
+          (network) => network.chainId === parseInt(networkId, 16),
+        );
+
+        const symbol = matchedChain?.nativeCurrency?.symbol ?? null;
```

Comment:
Just a suggestion to rename "symbol" to "matchedSymbol" because it is not the symbol of the token but the symbol we are trying to match in the list